platform/linuxbsd/libudev-so_wrap.h

```diff
@@ -0,0 +1,294 @@
+// This file is generated. Do not edit!
+// see https://github.com/hpvb/dynload-wrapper for details
+// generated by ./generate-wrapper.py 0.1 on 2021-02-17 10:38:58
+// flags: ./generate-wrapper.py --include /usr/include/libudev.h --sys-include <libudev.h> --soname libudev.so.1 --init-name libudev --output-header libudev.h --output-implementation libudev.c
+//
+#include <dlfcn.h>
+#include <stdio.h>
+#define gnu_dev_major gnu_dev_major_orig
```

Comment:
Should *probably* omit gnu_ prefixes for this one. It seems that this one comes from `/usr/include/sys/sysmacros.h` so we probably don't want these in the wrappers.

Reply:
Makes sense, fixed.

Reply:
BTW until now I thought `--omit-prefix` would strip a prefix from the symbols we do include, e.g. change `udev_new()` to `new()` if omitting `udev_`. Now I understand that it excludes symbols which start with this prefix. Might be worth clarifying in the docs.

Reply:
Right, I'll clarify the help screen